/mnt/phd/jihu/VaxDemandDistance/utils/evaluate_model.py:63: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  selected_locations['id'] = range(selected_locations.shape[0])
_10000_1_4q_mnl_flexthresh2_3_15_logdistabove1.0
Evaluating random order FCFS with Chain type: Dollar; Model: MaxVaxDistLogLin; M = 5, K = 10000, R = None.
 Results stored at /export/storage_covidvaccine/Result/MaxVaxDistLogLin/M5_K10000_4q/Dollar

Start computing distdf...

Start constructing blocks...

Start assignments...

Computed abepsilon in: 1.422 seconds.
Assigning individuals...
time3 - time2: 27.13
time4 - time3: 306.466
Number of full locations: 762  out of  4035
